test/models/notice_test.rb

```diff
@@ -604,11 +604,11 @@
           end
 
           it "noticeにtagが紐付いていること" do
-            Notice.find(@notice).tags.first.name.must_equal @tag.name
+            Notice.find(@notice.id).tags.first.name.must_equal @tag.name
           end
 
           it "ほかのnoticeにもtagが紐付いていること" do
-            Notice.find(@other_notice).tags.map {|t| t.name }.join(",").must_include @tag.name
+            Notice.find(@other_notice.id).tags.map {|t| t.name }.join(",").must_include @tag.name
```

Comment:
Line is too long. [97/80]<br>Space between { and | missing.<br>Pass `&:name` as an argument to `map` instead of a block.